docs/Makefile.am

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) 2022 Cisco Systems, Inc.  All rights reserved.
+#
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+# Random notes:
+#
+# - It would have been nice to auto-generate some of the man page
+#   listing files (e.g., man-openmpi/man3/index.rst), but Sphinx does
+#   not handle having source files split across the source tree and
+#   build tree.  We could have sym linked everything into the build
+#   tree, but... we didn't.
+
+###########################################################################
+# JMS: to investigate
+
+# handle mpiext affinity and cuda man pages
+###########################################################################
+
+SPHINX_CONFIG      = conf.py
+TEXT_SOURCE_FILES  = $(srcdir)/license/*.txt
+IMAGE_SOURCE_FILES = $(shell find $(srcdir) -name \*.png)
```

Comment:
This is a GNUMake-ism.  And I don't love running find like this...

Reply:
Fair enough.  I did it this way because there's currently 800+ RST and PNG files (the majority of which are man pages), and I didn't want to try to maintain a list of them here in the Makefile.am.  That being said, I don't feel _too_ strongly about that; if you prefer that I put the list here in Makefile.am, I will.  What would you prefer?

Reply:
Actually, here's a better idea -- are you cool with this?

```
IMAGE_SOURCE_FILES = \
        $(srcdir)/openmpi_logo.png \
        $(srcdir)/installing-open-mpi/required-support-libraries-dependency-graph.png
RST_SOURCE_FILES   = \
        $(srcdir)/*.rst \
        $(srcdir)/building-apps/*.rst \
        $(srcdir)/developers/*.rst \
        $(srcdir)/faq/*.rst \
        $(srcdir)/features/*.rst \
        $(srcdir)/installing-open-mpi/*.rst \
        $(srcdir)/license/*.rst \
        $(srcdir)/man-openmpi/*.rst \
        $(srcdir)/man-openmpi/man*/*.rst \
        $(srcdir)/man-openshmem/*.rst \
        $(srcdir)/man-openshmem/man*/*.rst \
        $(srcdir)/networking/*.rst \
        $(srcdir)/news/*.rst \
        $(srcdir)/release-notes/*.rst \
        $(srcdir)/running-apps/*.rst
```

Reply:
I think wildcards will work across Make implementations.  I'm fine not listing 800 files explicitly, but we can't do it using GNU make-isms :).  If wildcards aren't portable across Make, then computing the set in configure might be our best (but awful) option?